docs/release_notes/flare_250.rst

```diff
@@ -143,3 +143,116 @@ Developer Tutorial Page
 To let users quickly learn Federated Learning with FLARE, we developed a `tutorial web page <https://nvidia.github.io/NVFlare>`_ with
 both code and video to interactively learn how to convert and run FL in a few minutes. We also
 created a tutorial catalog to help you easily search and find the examples you are interested in.
+
+**********************************
+Migration to 2.5.0: Notes and Tips
+**********************************
+
+FLARE 2.5.0 introduces some API and behavior changes. This migration guide will help you to migrate from the previous NVFlare version
+to the current version.
+
```

Comment:
should we mention that all "name" for app_opt need to be changed to the full "path" in config?